### DISCUSSION TITLE
How to debug the postgres driver

### DISCUSSION THREAD

**Discussion by OscarGA** [mtxr/vscode-sqltools#1214] [Q&A]
Hi everyone,
I am trying to debug the postgres driver to start contributing to this project but I am running into some problems. I have searched for documentation on the debugging part for this project, but I can't find anything related to this. 
Can anyone help me with the introductory steps to debug the project?

**Comment by gjsjohnmurray** [comment]
@OscarGA are you still looking for help with this?
